plugins/azure/resoto_plugin_azure/resource/network.py

```diff
@@ -3487,7 +3491,7 @@ class AzureFirewallPolicy(AzureResource):
         >> Bend(AzureFirewallPolicyIntrusionDetection.mapping),
         "provisioning_state": S("properties", "provisioningState"),
         "rule_collection_groups": S("properties") >> S("ruleCollectionGroups", default=[]) >> ForallBend(S("id")),
-        "size_string": S("properties", "size"), # TODO: check if this is actually an int?
+        "size_string": S("properties", "size"),  # TODO: check if this is actually an int?
```

Comment:
This property also have a `string` type

https://learn.microsoft.com/en-us/rest/api/virtualnetwork/firewall-policies/list-all?tabs=HTTP#firewallpolicy



properties.size | string | A read-only string that represents the size of the FirewallPolicyPropertiesFormat in MB. (ex 0.5MB)
-- | -- | --